northstar/src/runtime/island/seccomp.rs

```diff
@@ -35,38 +48,66 @@ mod bindings {
     pub const SECCOMP_RET_LOG: u32 = 2147221504;
```

Comment:
@inorick Do you remember why we don't get `SECCOMP_RET_LOG` is not included in the generated bindings?

Reply:
I think you added that in fb430ec51dd34bd271bcae60a4d3e0f75ccfc77b.

I do not know more than what the comment says:
> SECCOMP_RET_LOG is implemented by Linux 4.14 but not present in all C libs